### PR TITLE
✨ OP Bedrock upgrade notification

### LINKED ISSUE #1060: Show warning for Bedrock update on Optimism
At least during the update: https://www.optimism.io/bedrock-upgrade

June 6, 2023 at 16:00 UTC.

The upgrade will require 2-4 hours of downtime for OP Mainnet.

### PR DESCRIPTION
**Pull request by franm91**
Added topbar to notify users about the OP upgrade.
closes #1060 

image
image

### CODE REVIEW

**Review by jgalat — APPROVED** [review]
lgtm, just a minor comment

File: components/Topbar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const handleClose = () => {
+    setShowBar(false);
+  };
+
+  return showBar && displayNetwork.id === 10 ? (
+    <Box
+      sx={{
+        display: 'flex',
+        alignItems: 'center',
+        justifyContent: 'space-between',
```
[the remark this answers is not in the document]
Comment (JuampiRombola) [reply components/Topbar/index.tsx:27]: it has 2 children, the text and the close button 🤔

File: components/Topbar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        bgcolor: palette.orange,
+        color: 'white',
+        height: { xs: '64px', sm: '32px' },
+        width: '100%',
```
Comment (JuampiRombola) [inline components/Topbar/index.tsx:31]:
``` suggestion
        width: '100%',
        p: 1,
```

File: i18n/es/translation.json [hunk 3]
```diff
@@ -261,5 +261,7 @@
   "The fixed borrowing interest APR at current utilization level.": "La TNA fija para préstamo en el nivel actual de utilización.",
   "Make sure to repay your fixed borrows before {{date}} to avoid penalty fees.": "Asegúrese de repagar sus préstamos a tasa fija antes del {{date}} para evitar multas",
   "Reserve ETH for gas fees.": "Reserva ETH para el costo de la transacción.",
-  "Market APR": "TNA del Mercado"
+  "Market APR": "TNA del Mercado",
+  "OP Mainnet upgrade to Bedrock release: June 6, 2023, 16:00 UTC. 2-4 hours downtime expected.": "Actualización de OP Mainnet a la versión Bedrock: 6 de junio de 2023, 16:00 UTC. Se espera un tiempo de inactividad de 2 a 4 horas.",
```
[the remark this answers is not in the document]
Comment (franm91) [reply i18n/es/translation.json:265]: It's in the same format as the official OP report.

**Review by JuampiRombola — APPROVED** [review]

**Review by jgalat — APPROVED** [review]

**Pull request merged**